capstone/src/main/webapp/scripts/poll.js

```diff
@@ -26,16 +27,49 @@ function getPollOptions() {
       const optionsContainer = document.getElementById("options-container");
       optionsContainer.innerHTML = "";
       let maxVotes = getMaxVotes(pollData);
+      topChallenge = pollData["options"][0]["text"];
       pollData["options"].forEach(option => {
         renderOptionElement(option, maxVotes);
       });
       return pollData["votedOptions"];
     })
     .then(votedOptions => {
       handleCheck(votedOptions);
+    })
+    .then(checkWeek);
+}
+
+function checkWeek() {
+  getChallenge();
```

Comment:
What's the purpose of getChallenge here? Please correct me if I'm wrong, but unless it's relying on the weeklyChallenge element (which I also don't see it is), I don't see a need for it here?

Reply:
I changed the code in the new pull request so now checkWeek doesn't have getChallenge

Reply:
Okie dokie!